perma_web/perma/views/user_management.py

```diff
@@ -1357,7 +1361,7 @@ def limited_login(request, template_name='registration/login.html',
     """
 
     if request.method == "POST" and not request.user.is_authenticated:
-        username = request.POST.get('username')
+        username = request.POST.get('username', '').lower()
         try:
             target_user = LinkUser.objects.get(email=username)
```

Comment:
This type of scenario always makes me unnecessarily nervous, and I tend to throw before doing a db lookup if `username` is not in the format I expect. Mostly a druther.

Reply:
This makes so much sense. You know what? I want to audit the whole app, and see where else we do this. I think I'll make a separate ticket rather than addressing here, since as is, this basically keeps the same behavior as before (switching to looking up `''` instead of `None`)